cryodrgn/protocols/protocol_base.py

```diff
@@ -237,7 +237,7 @@ def _createParticleSet(self):
         Create a set of particles with the associated z_values
         :return: a set of particles
         """
-        inputImgSet = self.inputParticles.ptcls.getExtended()
+        inputImgSet = self.inputParticles.get().ptcls.get()
```

Comment:
self.inputParticles is a Pointer as usual with any input param (sets).

ptcls is again a pointer.... so we also need get.  Optionally, other sets like set of particles have a getter like getParticles () that hides the .get() of the pointer.